js/components/MagnetMapView.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Dimensions,
+  StyleSheet,
+} from 'react-native';
+import MapView from 'react-native-maps';
+
+export default class MagnetMapView extends Component {
+  render() {
+    const { height, width } = Dimensions.get('window');
```

Comment:
I don't think the mapview with always be fullscreen. Is there a way that the map view can size to fit it's parent?